Objects/Database/ShapeSets/scriptedparts.json

```diff
@@ -1409,7 +1409,7 @@
 			"box": { "x": 2, "y": 2, "z": 2 },
 			"density": 250.0,
 			"physicsMaterial": "Metal",
-			"scripted": { "filename": "$MOD_DATA/Scripts/interactable/ToneGenerator.lua", "classname": "ToneGenerator" },
+			"scripted": { "filename": "$MOD_DATA/Scripts/interactable/ToneGenerator.lua", "classname": "totegenerator" },
```

Comment:
"totegenerator" does not follow our naming guidelines

Reply:
https://github.com/SMTheGuild/The-Modpack/pull/25/commits/6147275cb011e3b6040ef52041d2dcab6a98da6a